### PR TITLE
str #20144 UnzipWith pushed twice when sinks were closing

### LINKED ISSUE #20144: Cannot push port twice in UnzipWith
Relevant stacktrace snippet:

```
at akka.stream.scaladsl.UnzipWith2$$anon$1$$anon$2.onPush(UnzipWithApply.scala:366)
at akka.stream.stage.GraphStageLogic.push(GraphStage.scala:439) 
at scala.Predef$.require(Predef.scala:219) 
```

### PR DESCRIPTION
**Pull request by johanandren**
Fixes #20144

### CODE REVIEW

**Comment by drewhk** [comment]
Great! this should be backported.

File: akka-stream/src/main/boilerplate/akka/stream/scaladsl/UnzipWithApply.scala.template [hunk 1]
```diff
@@ -69,6 +72,7 @@ class UnzipWith1[In, [#A1#]](unzipper: In ⇒ ([#A1#])) extends GraphStage[FanOu
     [#setHandler(out0, new OutHandler {
       override def onPull() = {
         pendingCount -= ##1
+        pending0 = false
```
Comment (ktoso) [inline akka-stream/src/main/boilerplate/akka/stream/scaladsl/UnzipWithApply.scala.template]: Ignore this for now: would be fun to instead use a long with bitmasks

**Comment by ktoso** [comment]
LGTM

**Comment by drewhk** [comment]
LGTM

**Comment by pkinsky** [comment]
Awesome, thanks for the quick fix

**Pull request merged**